test/basic_parser.cpp

```diff
@@ -357,6 +357,8 @@ class basic_parser_test
 
         // control after escape
         bad ("\"\\\\\n\"");
+
+        good(R"json("Hello\u0000World")json"); // Issue 38
```

Comment:
If you're going to mention the issue, please make it a hyperlink before the statement.

However, since this library has not yet been released, it is not important to even mention the issue. Your call.

Reply:
replaced with comment to indicate the nature of the test rather than the origin